docker/Dockerfile.nvidia

```diff
@@ -27,28 +27,27 @@ ENV DEBIAN_FRONTEND noninteractive
 # nodesource: nvdashboard requires nodejs>=10  
 RUN apt-get update -y && \
     apt-get install -y -q \
-        apt-utils \
-        vim \
-        curl \
-        mpich libmpich-dev && \    
+        apt-utils curl\
+#        mpich libmpich-dev \    
+        vim && \
     curl -sL https://deb.nodesource.com/setup_12.x | bash - && \
     apt-get install -y -q \
         nodejs \
         liblapack-dev  \
         libblas-dev \
         libibverbs-dev libmlx4-1 libmlx5-1 ibutils \
         texlive-latex-extra texlive-fonts-recommended dvipng cm-super \
-        python-dev python3-dev python3-venv && \
+        python-dev python3-dev python3-venv ffmpeg && \
```

Comment:
since this image inherits from python3.6 (actually: we could bump it to at least 3.7 or maybe also 3.8!) , we shouldn't need `python-dev python3-dev python3-venv`